utils/src/main/scala/cromwell/pipeline/utils/ApplicationConfig.scala

```diff
@@ -8,7 +8,7 @@ sealed trait ConfigComponent
 
 final case class WebServiceConfig(interface: String, port: Int) extends ConfigComponent
 
-final case class GitLabConfig(url: String, idPath: String, token: Map[String, String]) extends ConfigComponent
+final case class GitLabConfig(url: String, idPath: String, token: Map[String, String], gitlabDefaultFileVersion: String) extends ConfigComponent
```

Comment:
Maybe we should name it as `defaultFileVersion`? It is kinda obvious that we are talking about `GitLab` inside of `GitLabConfig` :)

Reply:
ok :)